pkg/scyllaclient/client.go

```diff
@@ -160,6 +160,23 @@ func (c *Client) Status(ctx context.Context, host string) (NodeStatusInfoSlice,
 	return all, nil
 }
 
+func (c *Client) LocalHostid(ctx context.Context, host string, retry bool) (string, error) {
```

Comment:
```suggestion
func (c *Client) LocalHostID(ctx context.Context, host string, retry bool) (string, error) {
```

Reply:
the pattern of using verbs isn't well established here but `GetLocalHostID` would be a pretty name

Reply:
I'll leave the naming to @zimnx then, but I'd like at least the CamelCase

Reply:
i'm fine with Get prefix, lets use it